[2025-03-30T01:45:50.014+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-30T01:45:50.182+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T01:45:47.604316+00:00 [queued]>
[2025-03-30T01:45:50.194+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T01:45:47.604316+00:00 [queued]>
[2025-03-30T01:45:50.233+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-30T01:45:50.257+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-03-30 01:45:47.604316+00:00
[2025-03-30T01:45:50.271+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1135) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-30T01:45:50.273+0000] {standard_task_runner.py:72} INFO - Started process 1137 to run task
[2025-03-30T01:45:50.274+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_data_pipeline', 'fetch_data', 'manual__2025-03-30T01:45:47.604316+00:00', '--job-id', '363', '--raw', '--subdir', 'DAGS_FOLDER/Gestioncommands.py', '--cfg-path', '/tmp/tmp437f4kwp']
[2025-03-30T01:45:50.276+0000] {standard_task_runner.py:105} INFO - Job 363: Subtask fetch_data
[2025-03-30T01:45:50.340+0000] {task_command.py:467} INFO - Running <TaskInstance: ecommerce_data_pipeline.fetch_data manual__2025-03-30T01:45:47.604316+00:00 [running]> on host 778b271a0247
[2025-03-30T01:45:50.442+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='ecommerce_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T01:45:47.604316+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T01:45:47.604316+00:00'
[2025-03-30T01:45:50.444+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-30T01:45:50.445+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-30T01:45:50.446+0000] {logging_mixin.py:190} INFO - Current task name:fetch_data state:running start_date:2025-03-30 01:45:50.183127+00:00
[2025-03-30T01:45:50.447+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline and current dag run status:running
[2025-03-30T01:45:50.448+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-30T01:45:50.449+0000] {Gestioncommands.py:40} INFO - Fetching data from API...
[2025-03-30T01:45:51.081+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host 'my.api.mockaroo.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-03-30T01:45:51.952+0000] {Gestioncommands.py:45} INFO - Data fetched successfully
[2025-03-30T01:45:51.956+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-30T01:45:51.971+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-30T01:45:51.972+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ecommerce_data_pipeline, task_id=fetch_data, run_id=manual__2025-03-30T01:45:47.604316+00:00, execution_date=20250330T014547, start_date=20250330T014550, end_date=20250330T014551
[2025-03-30T01:45:52.006+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-30T01:45:52.007+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-30T01:45:52.007+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline queued_at:2025-03-30 01:45:47.620026+00:00
[2025-03-30T01:45:52.008+0000] {logging_mixin.py:190} INFO - Task hostname:778b271a0247 operator:PythonOperator
[2025-03-30T01:45:52.022+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-30T01:45:52.071+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T01:45:52.074+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
